web/impact/impact/settings.py

```diff
@@ -361,7 +361,9 @@ class Base(Configuration):
             },
         }
     }
-
+    NO_REPLY_EMAIL = "noreply@masschallenge.org"
+    BCC_EMAIL = "systemlogbcc@masschallenge.org"
+    EMAIL_BACKEND = "mc.email_backends.AccelerateEmailBackend"
```

Comment:
The Python path in this string will not be reachable from impact. Did you mean to reference the new `ImpactEmailBackend`?

Reply:
(A second question comes to mind -- the fact that this bad setting didn't foil the test suggests maybe the test suite has a hole in it? I.e. if somebody were to break email sending by messing up `EMAIL_BACKEND` in the future, can we have a test that would alert us?)

Reply:
Thanks for catching that. 
Not sure what sort of test that would be. Any suggestions?

Reply:
I don't know offhand - if I'm correct in what I think we saw here, having an unusable `EMAIL_BACKEND` value did not cause the `EmailMultiAlternatives.send()` method to throw an error; this may be due to the monkeypatching that happens with email under the test runner, I don't know.

Perhaps a more direct angle would be, let's make sure we have a test that we know exercises `ImpactEmailBackend`.

Reply:
Will try to come up with a test that exercises the backend... it's a little tricky though...